protocol/0057-TRAN-transfers.md

```diff
@@ -87,6 +87,41 @@ To be able to dispatch rewards to reward pools of multiple markets pro rata to t
 - `reward metric asset` — (the settlement asset of all markets that will be in scope for the transfer)
 - `market scope` — a subset of markets in which parties are eligible to be rewarded from this transfer.
   - If the market scope is not defined / an empty list, it is taken as all the markets that settle in the reward metric asset.
+
+A party can further control their recurring transfer funding the reward pool by defining the entities which are within scope. Entities within scope can be either individual parties or [teams](./0083-RFPR-on_chain_referral_program.md#glossary). When scoping individuals, a subset of keys can be detailed, and when scoping teams a specific set of team ids can be detailed.
```

Comment:
> When scoping individuals, a subset of keys can be detailed
I don't see how this is possible. Seems like with individuals you can only choose between the 3 options of everybody, everybody that's part of a team, everybody that's not part of a team.